include/fastdds/dds/xtypes/dynamic_types/DynamicType.hpp

```diff
@@ -154,7 +155,7 @@ class DynamicType : public std::enable_shared_from_this<DynamicType>
 
     /**
      * State comparison according with the [standard] sections \b 7.5.2.8.4
-     * @param[in] other @DynamicType reference to compare to
+     * @param[in] other DynamicType reference to compare to
```

Comment:
```suggestion
     * @param[in] other @ref DynamicType reference to compare to
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)